cluster-scope/bundles/cert-manager/kustomization.yaml

```diff
@@ -2,7 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base/cert-manager.io/clusterissuers
-  - ../../base/cert-manager.io/issuers/ingress-letsencrypt-production
-  - ../../base/cert-manager.io/issuers/ingress-letsencrypt-staging
```

Comment:
do you still want to keep them under base?

Reply:
sorry I wasn't clear. My point is can we just delete those manifests then if we aren't using them at all?

Reply:
Oh, right, yes. Good catch.